code_schemes/somalia_operator.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Please consider adding to the standards repo

Reply:
will do